proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpTraceUtils.java

```diff
@@ -86,6 +88,30 @@ public class OtlpTraceUtils {
         }
       };
 
+  public static final Set<String> OTEL_METRICS =
+      ImmutableSet.of(
+          "process.runtime.jvm.threads.count",
+          "process.runtime.jvm.memory.usage",
+          "runtime.jvm.gc.count");
+
+  public static boolean isOtelMetric(String metricName) {
+    return OTEL_METRICS.contains(metricName);
+  }
+
+  public static String getAttrValByKey(List<KeyValue> attributesList, String key) {
+    Optional<KeyValue> sourceAttr =
+        attributesList.stream().filter(kv -> key.equals(kv.getKey())).findFirst();
+
+    return sourceAttr.map(keyValue -> fromAnyValue(keyValue.getValue())).orElse(null);
```

Comment:
I haven't figured out where fromAnyValue() is defined.

Reply:
It is another private method defined in the OtlpTraceUtils.java